client/http_sender.go

```diff
@@ -29,10 +29,11 @@ import (
 
 	"golang.org/x/net/context"
 
-	"github.com/cockroachdb/c-snappy"
+	snappy "github.com/cockroachdb/c-snappy"
```

Comment:
My go imports for whatever reason removes this line and then it won't compile. How do I fix that?